composer.json

```diff
@@ -74,6 +76,7 @@
         "ci:php:sniff": "phpcs config src tests",
         "ci:php:fixer": "php-cs-fixer --config=config/php-cs-fixer.php fix --dry-run -v --show-progress=dots --diff-format=udiff config/ src/ tests/",
         "ci:php:md": "phpmd src text config/phpmd.xml",
+        "ci:php:psalm": "psalm --show-info=false",
```

Comment:
Had overlooked this, was running `composer run ci` locally.

There's also vimeo/psalm#1192  to take into account, the fix for which wasn't yet introduced in 3.2 (which you're stuck using because of various dependencies).

Reply:
OK.  Let's do this as a separate PR.  I've added #790 so that it is captured and this conversation can be resolved.

Reply:
Oh, I see you've pushed a commit to this PR adding the step to Travis, but it's currently failing due to some remaining issues.  I suggest also addressing the remaining issues as that (or another) separate PR - this one is beginning to become hard to follow and keep track of via the GitHub UI.